sessionctx/variable/error.go

```diff
@@ -32,4 +33,10 @@ var (
 	ErrTruncatedWrongValue         = dbterror.ClassVariable.NewStd(mysql.ErrTruncatedWrongValue)
 	ErrMaxPreparedStmtCountReached = dbterror.ClassVariable.NewStd(mysql.ErrMaxPreparedStmtCountReached)
 	ErrUnsupportedIsolationLevel   = dbterror.ClassVariable.NewStd(mysql.ErrUnsupportedIsolationLevel)
+	errUnknownSystemVariable       = dbterror.ClassVariable.NewStd(mysql.ErrUnknownSystemVariable)
+	errValueNotSupportedWhen       = dbterror.ClassVariable.NewStdErr(mysql.ErrNotSupportedYet, pmysql.Message("%s = OFF is not supported when %s = ON", nil))
+	// ErrFunctionsNoopImpl is an error to say the behavior is protected by the tidb_enable_noop_functions sysvar.
+	// This is copied from expression.ErrFunctionsNoopImpl to prevent circular dependencies.
+	// It needs to be public for tests.
+	ErrFunctionsNoopImpl = dbterror.ClassVariable.NewStdErr(mysql.ErrNotSupportedYet, pmysql.Message("function %s has only noop implementation in tidb now, use tidb_enable_noop_functions to enable these functions", nil))
```

Comment:
would `s/use tidb_enable_noop_functions/set tidb_enable_noop_functions to true/` friendlier?

Reply:
I copied this from `expression.ErrFunctionsNoopImpl` where it's an existing error message. I agree with changing it; actually to "SET tidb_enable_noop_functions=1` because this is an exact command. But I would like to do this in both places in a followup PR if that's okay with you.